src/main/java/com/google/devtools/build/lib/buildeventstream/proto/build_event_stream.proto

```diff
@@ -543,6 +543,7 @@ enum TestStatus {
   REMOTE_FAILURE = 6;
   FAILED_TO_BUILD = 7;
   TOOL_HALTED_BEFORE_TESTING = 8;
+  SKIPPED = 9;
```

Comment:
Following up: can we just leverage this existing status? https://github.com/bazelbuild/bazel/blob/5de680aa3b9d99c37954447e6ca7b66d1b551978/src/main/java/com/google/devtools/build/lib/buildeventstream/proto/build_event_stream.proto#L272

For context, that was invented for the precursor to the work you're doing (cpu-specific skipping)

Reply:
If I remove  `SKIPPED` from the `TestStatus` enum, what `TestStatus` value should I use instead? `NO_STATUS` That seems like the closest to "skipped".

Reply:
I honestly don't know. Can you see how the prior precedent for cpu-based skipping does it? I think we can just do whatever it's doing.

I'm happy to help look at that if you're not sure where to look. Let me know.

Reply:
All right, I dug in more. I agree this is somewhere where results aren't communicated well. I asked the BES expert for advice. Stay tuned...

Reply:
Following up on the event protocol expert feedback, their advice is:

> "I think there is nothing wrong with SKIPPED TestStatus as a concept, but the text of the PR describes a target that was not built, which should be reported on TargetComplete event instead of TestResult/TestSummary."

If we want to honor that advice (which I think is fair), I see two options:

1) Trigger the existing `Aborted.SKIPPED` message. It's not ideal but it's an already-understood precedent.
2) Switch that to a new failure entry in the `TargetComplete` message.

I think either way we'd need extra code for Bazel to properly report the result to the command line, since the current result reads `TestStatus`. I prototyped some Bazel code that connects the dots without the help of `TestStatus`. Check it out at https://github.com/bazelbuild/bazel/compare/master...gregestren:skipped_status?diff=split&expand=1.

I touches a few files but the logic isn't so bad: mostly passing the `skipped` bit through a few classes until it gets to the code responsible for reporting.

Reply:
Wow. Your prototype worked really well. I took that and added a few comments, but otherwise it looks complete. Thanks!

With that I was able to remove the `SKIPPED` test status.